setup.py

```diff
@@ -210,6 +210,11 @@ def package_files(prefix: str, directory: str, extension: str):
     'oci>=2.88.2,<3.0.0',
 ]
 
+extra_deps['gcs'] = [
+    'google-cloud-storage==2.10.0',
```

Comment:
do you think we can pin these more loosely?

Reply:
Good point! Looking at the API changelog, it's probably alright with >=2.0.0.